src/components/CouponDetails/index.jsx

```diff
@@ -483,11 +490,15 @@ class CouponDetails extends React.Component {
 
     this.resetCodeActionStatus();
 
-    if (stateKey) {
+    if (stateKey || doesCodeActionHaveErrors) {
```

Comment:
We can only check one variable, if `stateKey` is `null` so would be `doesCodeActionHaveErrors`.